src/vega.schema.ts

```diff
@@ -455,6 +455,11 @@ export interface VgLegendConfig extends VgLegendBase {
   labelFontSize?: number;
 
   /**
+   * Maximum allowed pixel width of axis tick labels.
+   */
+  labelLimit?: number;
```

Comment:
I test those config and they don't work. I have asked Jeff on slack and still waiting for his response.